<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget6</class>
 <widget class="QWidget" name="MyWidget6">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label4_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>左上角经度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>日范围：2 -- 8    时范围：0 -- 23</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>矩形区域2：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit4_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label7">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit4_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>纬度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>纬度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>116.2 -- 116.6, 39.8 -- 40.0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit4_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>39.89</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>左上角经度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>矩形区域1：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit2_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>116.5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit3_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>40</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit2_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>116.45</string>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>时 --</string>
   </property>
  </widget>
  <widget class="QLabel" name="label7_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>右下角经度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>右下角经度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>纬度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>北京市中心经纬度大致范围：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label6_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>时</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>116.3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2008年2月</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>纬度：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit3_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>39.92</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit4_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>输入k:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>291</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询往来矩形区域1,2的k条频繁路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit2_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>116.35</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit3_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>39.85</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
